stl/inc/chrono

```diff
@@ -282,14 +281,15 @@ template <class _Clock, class _Duration>
 _INLINE_VAR constexpr bool _Is_trivially_swappable_v<chrono::time_point<_Clock, _Duration>> =
     _Is_trivially_swappable_v<_Duration>;
 
-template <intmax_t _Ax, intmax_t _Bx>
-struct _Lcm : integral_constant<intmax_t, (_Ax / _Gcd<_Ax, _Bx>::value) * _Bx> {}; // compute LCM of _Ax and _Bx
+_NODISCARD constexpr intmax_t _Lcm(intmax_t _Ax, intmax_t _Bx) {

```

Comment:
Ditto needs `noexcept` and top-level `const` parameters

Reply:
Maybe also add a comment explaining what this function does? (Mimicking other functions)